### ISSUE TITLE
Non a11y conformant background color used in email template

### ISSUE THREAD

**Issue by SubandiK** [rails/rails#53828]
### Steps to reproduce
[Look at the code](https://github.com/rails/rails/blob/main/railties/lib/rails/templates/rails/mailers/email.html.erb#L35) or try it in an accessibility contrast testing tool:
We use the color #7f7f7f which does not pass when checked vs #FFFFFF.

This only has a contrast ratio of 4:1, which does not pass accessibility tests

### Expected behavior
Email template should be accessible. The color #757575 could be used which looks very similar but has a contrast ratio of 4.6:1 and would pass most contrast checks.

### Actual behavior
Email template uses potentially inaccessible color for background which can have white letters.

### System configuration
irrelevant

**Comment by rafaelfranca** [comment]
Thank you for the issue. That template is just for the a mailer preview page. Not to say that it doesn't need to be fixed, so feel free to send a PR. I'm closing since this isn't a top priority issue.

**Comment by SubandiK** [comment]
Happy to submit a PR, was not clear about the rules and thought creating an issue first would be better than not-asked-for PRs.
